tsconfig.config.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.node.json",
+  "include": ["vite.config.*", "vitest.config.*", "cypress.config.*", "playwright.config.*"],
```

Лишнего сгенерировало.

```suggestion
  "include": ["vite.config.*", "vitest.config.*"],
```

---

Файл `vite.config.ts`, который подключается здесь, в ветке вижу. Файл `vitest.config.ts` отсутствует, хотя он должен быть. Видимо, он потерялся!